src/xpk/core/core.py

```diff
@@ -1406,6 +1617,35 @@ def run_gke_node_pool_create_command(
       xpk_print(f'Delete Nodepools returned ERROR {max_return_code}')
       return 1
 
+  # Enable Workload Identity on existing Nodepools
```

Comment:
nit to add comment saying the xpk doesn't support disabling WI on clusters that have it enabled already? (Which makes sense, not saying we should add this)

Reply:
done